solidity/dashboard/src/connectors/explorer-mode-connector.js

```diff
@@ -0,0 +1,40 @@
+import { AbstractConnector } from "./abstract-connector"
+import { WALLETS } from "../constants/constants"
+import { getWsUrl } from "./utils"
+import Web3ProviderEngine from "web3-provider-engine"
+import WebsocketSubprovider from "web3-provider-engine/subproviders/websocket"
+import ExplorerModeSubprovider from "./explorerModeSubprovider"
+import { EventEmitter } from "events"
+
+export class ExplorerModeConnector extends AbstractConnector {
+  /** @type {string} To store selected account by user. */
+  selectedAccount = ""
+
+  constructor() {
+    super(WALLETS.EXPLORER_MODE.name)
+    this.eventEmitter = new EventEmitter()
+  }
+
+  setSelectedAccount = (address) => {
+    this.selectedAccount = address
```

Comment:
I think we should emit `accountsChanged` event here. Probably will be not used in our dapp via `ExplorerModeConnector`  but seems reasonable to emmit this type of event according to the rest connectors.